src/components/WwwFrame/Header/KvAtbModalContainer.vue

```diff
@@ -70,6 +86,27 @@ const modalPosition = computed(() => {
 	const top = `${headerPosition.bottom}`;
 	return { right, top };
 });
+
+const fetchUserData = async () => {
+	await apollo.query({
+		query: userAtbModalQuery,
+	}).then(({ data }) => {
+		userData.value = data;
+	}).catch(e => {
+		logFormatter(e, 'Modal ATB User Data');
+	});
+};
+
+onMounted(async () => {
+	await fetchUserData();
+
+	myKivaExperimentEnabled.value = getIsMyKivaEnabled(
+		apollo,
+		$kvTrackEvent,
+		userData.value?.my?.userPreferences,
+		userData.value?.my?.loans?.totalCount || 0,
```

Comment:
We don't want to fallback to `0` but allow `undefined` to be provided in case of API error. If we need to pass in `0` for guests, we should only pass for guests.

Reply:
For example, on TY we use `!this.isGuest ? data?.my?.loans?.totalCount : 1,`

Reply:
Technically if `userData.value?.my` is undefined is a guest user otherwise will be 0 if a new logged in user.

A difference with TY page is that you have lent instead that for ATB modal you haven't, you are 1 step before, technically 0 makes sense.

For this case, we are not using `!this.isGuest ? data?.my?.loans?.totalCount : 1` to show a different view as TY page, we are using this just to pass this condition here: https://github.com/kiva/ui/blob/a09349509d087419b6aa9e2d820e8c30167f494d/src/util/myKivaUtils.js#L144

Makes sense? Otherwise I can pass 1 as fallback, but doesn't makes sense to me as you haven't lent yet.

Reply:
Yeah makes sense, in that case `!this.isGuest ? data?.my?.loans?.totalCount : 0` might make sense, only using the fallback for guests.